modules/spo-binding-workloads.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * security/security_profiles_operator/spo-seccomp.adoc
+// * security/security_profiles_operator/spo-selinux.adoc
+
+ifeval::["{context}" == "spo-seccomp"]
+:seccomp:
+:type: seccomp
+:kind: SeccompProfile
+endif::[]
+ifeval::["{context}" == "spo-selinux"]
+:selinux:
+:type: SELinux
+:kind: SelinuxProfile
+endif::[]
+
+:_content-type: PROCEDURE
+[id="spo-binding-workloads_{context}"]
+= Binding workloads to profiles with ProfileBindings
+
+You can use the `ProfileBinding` resource to bind a security profile to the `SecurityContext` of a container.
+
+.Procedure
+
+. To bind a `Pod` that uses an `nginx:1.19.1` image to the `profile` example `{kind}` profile, create a `ProfileBinding` object in the same namespace with the `Pod` and the `{kind}` objects:
```

Comment:
Is the `profile` usage here redundant in wording?

```
... image to the `profile` example `{kind}` profile
```

to 

```
... image to the example `{kind}` profile
```

?

Reply:
Yes - thank you